addons/mail/models/res_users_settings.py

```diff
@@ -10,21 +13,59 @@ class ResUsersSettings(models.Model):
     is_discuss_sidebar_category_channel_open = fields.Boolean(string="Is discuss sidebar category channel open?", default=True)
     is_discuss_sidebar_category_chat_open = fields.Boolean(string="Is discuss sidebar category chat open?", default=True)
 
-    # RTC
+    # Voice
     push_to_talk_key = fields.Char(string="Push-To-Talk shortcut", help="String formatted to represent a key with modifiers following this pattern: shift.ctrl.alt.key, e.g: truthy.1.true.b")
     use_push_to_talk = fields.Boolean(string="Use the push to talk feature", default=False)
-    voice_active_duration = fields.Integer(string="Duration of voice activity in ms", help="How long the audio broadcast will remain active after passing the volume threshold")
+    voice_active_duration = fields.Integer(string="Duration of voice activity in ms", default=200, help="How long the audio broadcast will remain active after passing the volume threshold")
     volume_settings_ids = fields.One2many('res.users.settings.volumes', 'user_setting_id', string="Volumes of other partners")
 
+    # Video
+    use_blur = fields.Boolean(string="Use blur effect")
+    show_only_video = fields.Boolean(string="Show video participants only")
+
+    # Notification
+    custom_notifications = fields.Selection(
+        [("all", "All Messages"), ("no_notif", "Nothing")],
+        "Customized Notifications",
+        help="Mentions only if not specified. This setting will only applied to channels",
+    )
+    mute_until_dt = fields.Datetime(string='Mute notifications until', help="If set, the user will not receive notifications from all the channel until this date.")
+
+    @api.model
+    def _unmute(self):
```

Comment:
I was confused at first, I think this should have a more specific name to indicate this is the cron cleanup, and not unmute for the user member.